tests/flow/destructuring/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -458,27 +458,23 @@ function tup_pattern2<X>([proj]: Proj<X>) {} // proj: X
 function rest_antipattern<T>(...t: T) {} // nonsense
 function rest_pattern<X>(...r: X[]) {} // r: X[]
 
-function obj_rest_pattern<X>(
-  { _, ...o }: { _: any, x: X } // o: { x: X }
```

Comment:
However, in this case, the previous behavior was wrong in my opinion. The previous comment was at the end of the line and "attached" to the `{` but now it has somehow been moved to the first argument of the function.